avalon/tools/cbloader/app.py

```diff
@@ -168,9 +170,19 @@ def _assetschanged(self):
 
         asset_item = assets_model.get_active_index()
         if asset_item is None or not asset_item.isValid():
-            return
+            type = "asset"
```

Comment:
@mkolar could you put a comment here as to what this is achieving? Just looking at the code I'm having a hard time figuring it out how this solves it.

Also, to not shadow the `type` function I'd recommend just passing `"asset"` to `io.find_one` directly on line 178.

Reply:
Ah, wait... is this working in such a way that the loader still displays "silo" as tabs in the loader. Visually nothing changes, except that when you select nothing at all (no asset in selection) that you are operating on the 'silo asset'?

Reply:
> `Also, to not shadow the type function I'd recommend just passing "asset" to io.find_one directly on line 178.`

True. It doesn't make sence to store it into variable here.